components/data/buckets.tsx

```diff
@@ -0,0 +1,105 @@
+import IconGettingStarted from '../icons/GettingStarted';
+import IconGuide from '../icons/Guide';
+import IconMigration from '../icons/Migration';
+import IconSpec from '../icons/Spec';
+import IconTutorials from '../icons/Tutorials';
+import IconUseCases from '../icons/UseCases';
+import IconUsers from '../icons/Users';
+
+interface Bucket {
+  name: string;
+  title: string;
+  description: string;
+  link: string;
+  className: string;
+  borderClassName: string;
+  Icon: React.ComponentType<any>;
```

Comment:
Why this key is duplicated when there already exists an `icon` key with the same value?

Reply:
Instead of using `React.ComponentType<any>`, you can define a IconType in `types` folder as `React.ReactElement`.